src/Psalm/Internal/Analyzer/CommentAnalyzer.php

```diff
@@ -126,13 +126,13 @@ public static function getTypeFromComment(
                 $var_comment->deprecated = isset($comments['specials']['deprecated']);
                 $var_comment->internal = isset($comments['specials']['internal']);
                 if (isset($comments['specials']['psalm-internal'])) {
-                    $psalmInternal = reset($comments['specials']['psalm-internal']);
-                    if ($psalmInternal) {
-                        $var_comment->psalmInternal = $psalmInternal;
+                    $psalm_internal = reset($comments['specials']['psalm-internal']);
```

Comment:
reset is needed because the array $comments['specials']['psalm-internal'] doesn't start at zero - it starts at a different number. I didn't track down quite why that is.